notifications/rest/users/list-binding-of-user/list-binding.json.curl

```diff
@@ -0,0 +1,5 @@
+
+curl -X GET -G 'https://notify.twilio.com/v1/Services/ISXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX/User/00000001/Bindings' \
+    -d "StartDate=2017-01-17" \
```

Comment:
This probably is my mistake. Tag and StartDate makes no sense here. Can we remove it please?